pageserver/src/consumption_metrics.rs

```diff
@@ -252,6 +272,16 @@ pub async fn collect_metrics_task(
         ));
 
         // TODO add SyntheticStorageSize metric
+        let tenant_synthetic_size = tenant.calculate_synthetic_size().await?;
+        info!("tenant_synthetic_size: {}", tenant_synthetic_size);
+        current_metrics.push((
+            ConsumptionMetricsKey {
+                tenant_id,
+                timeline_id: None,
+                metric: ConsumptionMetricKind::SyntheticStorageSize,
+            },
+            tenant_synthetic_size,
+        ));
```

Comment:
This inline usage made me think of that we probably will always want a warning or even an error log message if the metric operation takes longer than the ticker timeout in the `tokio::select!`.

At least I think this might take a very long time for some tenants with a huge amount of layers, but that is still unknown. Regardless, we may want to also use the task_mgr::shutdown_watcher or cancellation token with this particular task.